backend/private-graph/graph/resolver.go

```diff
@@ -338,7 +340,7 @@ func (r *Resolver) isAdminInWorkspaceOrDemoWorkspace(ctx context.Context, worksp
 	} else {
 		workspace, err = r.isAdminInWorkspace(ctx, workspace_id)
 		if err != nil {
-			return nil, e.Wrap(err, "admin is not in workspace or demo workspace")
+			return nil, errors.New(AuthorizationError)
```

Comment:
could we keep this consistent with line 319? this should return err and we cleanup `isAdminInWorkspace` to only return an `AuthorizationError`, or else 319 should return `errors.New(AuthorizationError)`. just in case either of these are used elsewhere directly in a resolver.

Reply:
Good catch.
f3e549e4a513bcb770a2c0d83f8ec9c917b098c9